src/collections/_documentation/platforms/android/index.md

```diff
@@ -0,0 +1,597 @@
+---
+title: Android
+---
+{% capture __alert_content -%}
+This version of the Android SDK is in alpha. Sentry has been offering an official SDK for Android for years now. If you are looking for the stable LTS support of Sentry, please refer to the 1.x and its [docs]({%- link _documentation/clients/java/integrations.md -%}#android).
+{%- endcapture -%}
+{%- include components/alert.html
+    title="Note"
+    content=__alert_content
+    level="warning"
+%}
+
+## Integrating the SDK
+
+Sentry captures data by using an SDK within your application’s runtime. These are platform-specific and allow Sentry to have a deep understanding of how your app works.
+
+To install the Android SDK, please update your build.gradle file as follows:
+
+```groovy
+// ADD COMPATIBILITY OPTIONS TO BE COMPATIBLE WITH JAVA 1.8
+compileOptions {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+// ADD SENTRY ANDROID AS A DEPENDENCY
+dependencies {
+    implementation 'io.sentry:sentry-android:{version}'
+}
+```
+
+For other dependency managers, see the [central Maven repository](https://search.maven.org/artifact/io.sentry/sentry-android/1.7.28/jar).
```

Comment:
I don't understand what this means. Also, is this supposed to point to 1.7?

Reply:
^ @marandaneto @fdrabek

Reply:
I think that it should point to https://search.maven.org/artifact/io.sentry/sentry-android 
@bruno-garcia I think that it means that if you want to see syntax for Kotlin DSL, Bazel etc.. you should check it there.
Does it make sense?

Reply:
You want to link to the maven repositories we push the sdk? Android devs use more jcenter I believe at right now we had issues with maven central so I'm not sure the latest is going there /cc @marandaneto

Reply:
I'd just delete it till we get this published to maven central that has a nice way of showing how to integrate things using other DSLs.
let's focus on bintray/jcenter only right now.

Reply:
Oh I see what you mean. I wouldn't leave this in. It's OK to just have the groovy one for now. We can add the Lang selector later and show it here.

I suggest we remove this bit

Reply:
Got it. Will remove.